dev/com.ibm.ws.org.apache.cxf.cxf.rt.transports.http.3.2/src/org/apache/cxf/transport/http/ReferencingAuthenticator.java

```diff
@@ -25,11 +25,10 @@
 import java.lang.reflect.Modifier;
 import java.net.Authenticator;
 import java.net.PasswordAuthentication;
-import java.security.AccessController;
-import java.security.PrivilegedAction;
+import org.apache.cxf.common.util.ReflectionUtil;
```

Comment:
Remove this change, looks like it might no longer apply to the new version of CXF.